M/Mesa_EGL/bundled/patches/wl_scanner-no-native.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/meson.build b/meson.build
+index bca6b1f5c41..832db46e40e 100644
+--- a/meson.build
++++ b/meson.build
+@@ -1961,7 +1961,7 @@ endif
+ # TODO: symbol mangling
+ 
+ if with_platform_wayland
+-  dep_wl_scanner = dependency('wayland-scanner', native: true)
```

Comment:
Uhm, why is this patch necessary?  If they need a native `wayland-scanner`  isn't the solution to add `HostBuildDependency("Wayland_jll")`?  We do that already in https://github.com/JuliaPackaging/Yggdrasil/blob/061fca6012b0aa988834ae1bd427e088911b392b/G/GTK3/build_tarballs.jl#L76-L77 https://github.com/JuliaPackaging/Yggdrasil/blob/7b2b4526b40bd5933134958cf065557ff13bf8a5/G/GTK4/build_tarballs.jl#L94-L95 https://github.com/JuliaPackaging/Yggdrasil/blob/fcf41ad49508261d48048a5fddd2c6ff758494e1/W/Wayland_protocols/build_tarballs.jl#L34

Reply:
This and the FIXME were there because meson had problems with choosing to use the provided `wayland-scanner` from `HostBuildDependency("Wayland_jll")`; I don't remember the error, but it was cryptic and unhelpful :smile: I've reverted that change, and am trying to figure out the build failure.